src/main/java/com/ensao/gi5/lint/rules/LocalVariableRule.java

```diff
@@ -0,0 +1,42 @@
+package com.ensao.gi5.lint.rules;
+
+import com.ensao.gi5.lint.constantes.Constantes;
+import com.ensao.gi5.lint.rules.violations.Violation;
+import com.ensao.gi5.lint.visitor.LocalVariableVisitor;
+import com.ensao.gi5.lint.visitor.TypeNamingVisitor;
+import com.ensao.gi5.lint.wrapper.CompilationUnitWrapper;
+import com.ensao.gi5.lint.wrapper.LocalVariableWrapper;
+import com.ensao.gi5.lint.wrapper.TypeNamingWrapper;
+
+import java.util.LinkedHashSet;
+import java.util.Locale;
+import java.util.Set;
+
+public class LocalVariableRule extends Rule{
+
+    public LocalVariableRule(){
+        super(Constantes.LINT_REG_003, Level.HIGH);
+    }
+    @Override
+    public void apply(CompilationUnitWrapper compilationUnit) {
+        Set<LocalVariableWrapper> localVariableWrapperSet = new LinkedHashSet<LocalVariableWrapper>();
+        compilationUnit.accept(new LocalVariableVisitor(),localVariableWrapperSet);
+
+        for(LocalVariableWrapper localVariableWrapper : localVariableWrapperSet){
+            String name = localVariableWrapper.toString();
+            if(name != name.toLowerCase()){
```

Comment:
comparing Strings using `=` or `!=` may lead to false result
cf https://www.linkedin.com/posts/mustapha-belmokhtar_java-string-comparison-activity-6916117993038958592-jWfh